x-pack/plugins/ml/server/routes/saved_objects.ts

```diff
@@ -220,4 +224,50 @@ export function savedObjectsRoutes({ router, routeGuard }: RouteInitialization)
       }
     })
   );
+
+  /**
+   * @apiGroup JobSavedObjects
+   *
+   * @api {get} /api/ml/saved_objects/delete_job_check Check whether user can delete a job
+   * @apiName DeleteJobCheck
+   * @apiDescription Check the users ability to delete jobs. Returns whether they are able
+   *                 to fully delete the job and whether they are able to remove it from
+   *                 the current space.
+   *
+   * @apiSchema (body) jobIdsSchema (params) jobTypeSchema
+   *
+   */
+  router.post(
+    {
+      path: '/api/ml/saved_objects/delete_job_check/{jobType}',
+      validate: {
+        params: jobTypeSchema,
+        body: jobIdsSchema,
+      },
```

Comment:
maybe it's worth putting everything in params, e.g. `'/api/ml/saved_objects/delete_job_check/{jobType}/{jobId}'`

Reply:
`jobIds` is an array of ids, for all other similar kibana endpoints we pass the ids as an array in the body (not including endpoints which are simple wrappers around es endpoints)

Reply:
ok, I recall some endpoints with comma-separated job ids